Lib/platform.py

```diff
@@ -1464,9 +1464,38 @@ def invalidate_caches():
 
 ### Command line interface
 
-if __name__ == '__main__':
-    # Default is to print the aliased verbose platform string
-    terse = ('terse' in sys.argv or '--terse' in sys.argv)
-    aliased = (not 'nonaliased' in sys.argv and not '--nonaliased' in sys.argv)
+def _parse_args(args: list[str] | None):
+    import argparse
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument("args", nargs="*", choices=["nonaliased", "terse"])
+    parser.add_argument(
+        "--terse",
+        action="store_true",
+        help=("return only the absolute minimum information needed to identify the "
+            "platform"),
```

Comment:
Either align to opening delimiter:
```suggestion
        help=("return only the absolute minimum information needed "
              "to identify the platform"),
```
Or I'd just go for Black style:

```suggestion
        help=(
            "return only the absolute minimum information needed "
            "to identify the platform"
        ),
```